docs/reference/src/literal-expressions.adoc

```diff
@@ -34,25 +34,17 @@ LITERAL_SUFFIX : XID_Start XID_Continue*
 
 A numeric literal can be written using three bases:
 
-1. Decimal literal starts with a decimal digit and then a mixture of decimal digits and underscores.
-2. Hexadecimal literal starts with the character sequence `0x` and continues as a mixture of hex
-   digits and underscores.
-3. Binary literal starts with the character sequence `0b` and continues as a mixture of binary
-   digits and underscores.
+1. Decimal literal starts with a decimal digit and then decimal digits.
```

Comment:
"decimal digit and the decimal digits" sounds weird.
What about "Decimal literals consist of only decimal digits"? Also, I would add the regex like here:
https://doc.rust-lang.org/reference/tokens.html#integer-literals